spec/triggers/gcp-pub-sub.md

```diff
@@ -0,0 +1,22 @@
+#  Google Cloud Platform‎ Pub/Sub
+
+This specification describes the `gcp-pub-sub` trigger.
+
+```yaml
+triggers:
+- type: gcp-pubsub
+  metadata:
+    subscriptionSize: "5"
```

Comment:
Nitpicking but maybe specify that this is optional and mention what the default value is?

Reply:
@tomkerkhove this is fixed now.

Reply:
Thanks!